tests/test_update.py

```
@@ -614,29 +777,15 @@ def test_get_versions_exception_path(
-            *args: Additional positional arguments forwarded by the function.
-            **kwargs: Additional keyword arguments forwarded by the function.
-
-        Raises:
-            TypeError: If a supplied argument has an unsupported type.
-        """
-        raise TypeError("boom")
-
-    monkeypatch.setattr(update_module, "dict_get", raise_type)
-    pv, pl, ps = ent._get_versions({})
+    pv, pl, ps = ent._get_versions({"firmware_update_info": []})
     assert pv is None and pl is None and ps is None
```

Ruff (0.15.18)

[warning] Assertion should be broken down into multiple parts

Break down assertion into multiple parts

(PT018)